Examples/Run/Vertexing/Pythia8/AdaptiveMultiVertexFinderExample.cpp

```diff
@@ -26,7 +26,7 @@ int main(int argc, char* argv[]) {
   Options::addRandomNumbersOptions(desc);
   Options::addPythia8Options(desc);
   ParticleSelector::addOptions(desc);
-  Options::addOutputOptions(desc);
+  Options::addOutputOptions(desc, Options::None);
```

Comment:
So, `None` means *only* output dir?

Reply:
Yes. It might be a bit unlucky wording... Maybe `DirOnly` or something like that is better.